src/presentation/components/team/Member.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+  <li class="member">
+    <div class="member-name">
+      {{ props.teamMember.user.name }}
```

Comment:
props are available in template
```suggestion
      {{ teamMember.user.name }}
```